tests/testthat/test-stat-align.R

```diff
@@ -48,19 +48,20 @@ test_that("alignment adjusts per panel", {
   # data into account (#5227)
 
   df <- data_frame0(
-    x = c(0, 1, 1000, 1001),
-    y = c(-1, 1, -1, 1),
-    g = c("A", "A", "B", "B")
+    x = c(0, 1, 1000, 1001, 0, 1, 1000, 1001),
+    y = c(-1, 1, -1, 1, -1, 1, -1, 1),
+    f = c("A", "A", "B", "B", "A", "A", "B", "B"),
+    g = c("a", "a", "b", "b", "c", "c", "d", "d")
```

Comment:
Had to ensure >=2 groups per panel here for this test to test the thing it is supposed to test.